migrate/migrate.go

```diff
@@ -17,6 +18,8 @@ import (
 	pipep "github.com/heetch/migrate/pipe"
 )
 
+var nowFunc = time.Now
```

Comment:
You could have this function return a timestamp instead of having to format it every time?